### DISCUSSION TITLE
Which Pi should I buy, and how much RAM?

### DISCUSSION THREAD

**Discussion by andrewmccafferty** [mcguirepr89/BirdNET-Pi#1260] [Q&A]
I've been running a kind of rough and ready system of my own that uses Birdnet, but my Pi (it's one of the small Pi Zeroes I think) isn't powerful to run it, so I instead have a clunky system that just records clips on my pi and then my laptop inside does the analysis.

Anyway, I want to buy a Pi that will cope with running BirdNET (and maybe the system here) but I couldn't figure out what spec I need. Can anybody advise?

**Comment by alexbelgium** [comment]
Hi, you should first try this repo, the code is more optimized and more up-to-date : https://github.com/Nachtzuster/BirdNET-Pi. Perhaps it would allow to run also the analysis on your Pi Zero thanks to the service optimizations that the owner did. Also, it is more up-to-date.

Regarding your question, currently i'm running it on a rpi4 with 8go ram and it uses 10% of RAM. So if you get a rpi4 with 4go ram it should be way more than enough. The UI is pretty responsive too. The only part not to compromise on is the sdcard - I only use "Samsung Pro Endurance" that never failed me.

If you want to go really crazy fast you could get a small x86 mini-pc based on N100 (for example - just bought one for 110€ with 8gb ram + ssd ; including tax & transport) from aliexpress. It's a price only slightly higher than rpi + case + supply + high quality sdcard.

**Comment by stevedee** [comment]
Obviously there is a cost implication, so I don't see why you need more than a Pi4 + 2GB RAM.
I'm using a 160GB USB hard drive rather than an SD card, and after 12 months of use its only half full.

**Comment by merkurmaniac** [comment]
Are there any good single board computers that might be better suited than a Rasp Pi4 ?
